app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -283,7 +272,7 @@ class NavigationDrawerTestActivityTest {
   }
 
   @Test
-  fun testNavigationDrawerTestActivity_withAdminProfile_openNavigationDrawer_checkAdministratorControlsDisplayed() { // ktlint-disable max-line-length
+  fun testNavigationDrawerTestActivity_openNavDrawer_adminControlsIsDisplayed() {
```

Comment:
Here & above: suggest differentiating between an admin profile and user profile (I don't think it's obvious that the default profile state is admin).

Reply:
done. 
Did some fixes due to char length ktlint issue
Thinking to create a starter issue to rename these files 

`NavigationDrawerTestActivity` -> `NavigationDrawerActivity`
`NavigationDrawerTestActivityTest` -> `NavigationDrawerActivityTest`

Reply:
Test rename sounds good, but the activity doesn't seem right--this is a test activity (we would never open it in the production app, for example). The reason the test name isn't quite right is because this isn't testing that the drawer test activity is correct, it's testing that the drawer integration itself is correct. I think a simple test rename is sufficient, but that's not really worth filing an issue over imo--can we just do it quickly?

Also, we should consider rethinking the test strategy for the drawer as a whole (the current tests depend on HomeActivity behavior--that's not ideal). I suggest filing a backlog issue for this, though it won't be suitable for starters.